src/pages/CheckoutComplete.page.js

```diff
@@ -0,0 +1,26 @@
+const { expect } = require('@playwright/test');
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+export class CheckoutCompletePage extends BaseSwagLabPage {
+    url = '/checkout-complete.html';
+
+    get thankYouMessage() { return this.page.locator('h2.complete-header'); }
+
+    get dispatchMessage() { return this.page.locator('div.complete-text'); }
+
+    get backHomeButton() { return this.page.locator('#back-to-products'); }
+
+    async verifyThankYouMessage() {
+        await expect(this.thankYouMessage).toBeVisible();
+        await expect(this.thankYouMessage).toHaveText('Thank you for your order!');
```

Comment:
better not to have expects in the page object
and text that we are checking now on a really low level in the test automation framework

Reply:
better to see that checks in the spec file